src/components/Footer.js

```diff
@@ -10,33 +10,48 @@ function Footer() {
       <Link to="/motive">
         <Motiv src="./img/Logo.jpg" alt="Motive" />
       </Link>
+      <Link to="/mixedDrinks">
+        <Cocktails src="./img/Pinnchen.jpg" alt="Cocktails and Shots" />
```

Comment:
If you don't get the image source path from a data array, I would recommend you import this as it is recommended by React: 
```
import cocktailImg from "./img/Pinnchen.jpg"
...
   <Cocktails src={cocktailImg} alt="Cocktails and Shots" />
```